src/test/groovy/cz/smarteon/loxone/app/LoxoneAppTest.groovy

```diff
@@ -42,9 +42,9 @@ class LoxoneAppTest extends Specification implements SerializationSupport {
         config.getControl(AlarmControl) == alarmControl
 
         where:
-        alarmControl << [new AlarmControl(), Stub(AlarmControl)]
```

Comment:
does this mean that Lombok-classes can't be mocked?

Reply:
No, this was my bad. I've had to remove the stub for Room & Category because those classes are marked as @Value what results in immutable final classes, and a final class cannot be stubbed.

Reply:
I'm not used to write Kotlin code at the moment, but the test complained the alarmControl had two params & coom & category didn't

Reply:
I've added @NonFinal to the Category & Room class so the class itself is not final which allows stubbing

Reply:
may be the stubs here are not very useful and it sounds like having the model classes final is a good idea => let's remove the stubbing and keep `@value`